nrf_rpc/nrf_rpc.c

```diff
@@ -907,12 +923,14 @@ static void wait_for_response(const struct nrf_rpc_group *group, struct nrf_rpc_
 	NRF_RPC_DBG("Waiting for a response");
 
 	do {
-		nrf_rpc_os_msg_get(&cmd_ctx->recv_msg, &packet, &len);
+		nrf_rpc_os_msg_get(&cmd_ctx->recv_msg, &cmd_ctx->mutex, &packet, &len);
 
-		NRF_RPC_ASSERT(packet != NULL);
+		if (packet == NULL) {
```

Comment:
Could you clarify in the `nrf_rpc_os_msg_get` documentation, that returning NULL is used for signaling the response timeout?